Tasks/Chef/task.json

```diff
@@ -0,0 +1,66 @@
+﻿{
+    "id": "B719DB6C-40A2-4F43-9AFF-827825BAECAE",
+    "name": "Chef",
+    "friendlyName": "Chef",
+    "description": "Deploy to Chef environments by editing environment attributes",
+    "category": "Deploy",
+    "visibility": [
```

Comment:
Why are we having visibility in  two different places? is this a merge issue which we didn't catch in subraman's PR?